models/evms/evms_aggregate_daily_usage.sql

```diff
@@ -0,0 +1,94 @@
+   {{
+  config(
+        tags = ['dunesql'],
+        schema = 'evms',
+        alias = alias('aggregate_daily_usage'),
+        materialized ='incremental',
+        file_format ='delta',
+        incremental_strategy='merge',
+        unique_key=['blockchain', 'block_date'],
+        partition_by=['blockchain', 'block_date'],
```

Comment:
Do you have any better partition strategy here?

Reply:
what would a good alternative be? I didn't have a major reason for picking these, other than them being the unique keys. But I would always read multiple chains at once.